tests/app/test_jinja_filters.py

```diff
@@ -258,10 +260,22 @@ def test_format_duration(duration, formatted_duration, app):
             {"maximum": {"value": 123456789012345678901123456789012345678901234567890}},
             None,
         ),
+        ({"maximum": {"value": {"identifier": "set-maximum", "source": "answers"}}}, 5),
+        (
+            {"minimum": {"value": {"identifier": "set-minimum", "source": "answers"}}},
+            15,
+        ),
     ),
 )
-def test_get_width_for_number(answer, width):
-    assert get_width_for_number(answer) == width
+def test_get_width_for_number(answer, width, app):
+    with app.app_context():
+        schema = QuestionnaireSchema({})
+        schema.min_and_max_map = {
+            "set-maximum": {"maximum": 5, "minimum": 0},
+            "set-minimum": {"maximum": 15, "minimum": 5},
+        }
+        g.schema = schema
+        assert get_width_for_number(answer) == width
```

Comment:
This doesn't take into account chained values does i.e i.e 3 needs 2 and 2 needs 1?